src/qt/editor/helpwidget.cpp

```diff
@@ -207,7 +208,7 @@ void link(T& factory, std::string_view id, Document::DocumentHandle& handle) {
     if (auto* fo = factory->getFactoryObject(id)) {
         auto& typeName = fo->getTypeName();
         std::string name{typeName.substr(0, typeName.find_first_of('<'))};
-        replaceInString(name, "class ", "");
+        name = util::demangle(name.c_str());
```

Comment:
there is no need to call demangle here is it already done in PropertyFactoryObjectTemplate